node/config/types.go

```diff
@@ -1109,3 +1110,13 @@ type FaultReporterConfig struct {
 	// rewards. This address should have adequate funds to cover gas fees.
 	ConsensusFaultReporterAddress string
 }
+
+type CurioAlerting struct {
+	// PagerDutyEventURL is URL for PagerDuty's Events API v2 URL. Events sent to this API URL are ultimately
+	// routed to a PagerDuty service and processed.
```

Comment:
+ // The default is sufficient for integration with the stock commercial Pager Duty company's service.